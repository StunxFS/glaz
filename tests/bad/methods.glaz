struct Suma {
  a: i32,
  b: i32
}

impl Suma {
  pub fn add(self) i32 {
    return self.a + self.b;
  }
}

fn main() {
  let x = Suma{a: 1, b: 4};
  x.sub();
}
