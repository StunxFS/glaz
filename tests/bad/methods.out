tests/bad/methods.glaz:14:4: error: type `Suma` has no method `sub`
     | 
  14 |   x.sub();
     |   ^^^^^^^
     = help: 1 possibility: `add`
glaz error: could not compile `methods`, aborting due to previous error.
